articles/141_static_remapping.md

```diff
@@ -324,8 +324,7 @@ The replacement must be a single token which will become the node's new name.
 
 Remapping rules are applied in the following order:
 
-1. Node name remapping
-1. Namespace remapping
+1. Namespace or Node name remapping
```

Comment:
so now name are namespace rules are applied in the order they appear, right?

It would be great to make that clear here

Reply:
> order they appear

This is not the case, they order doesn't matter.
Their prefix is always the node name before remapping, maybe that can be made more clear.

Reply:
> This is not the case, they order doesn't matter.

They're still ordered within a rule, right? If I give `talker:__node:=foo` and `talker:__node:=bar`, then is the final name is `foo`?

Reply:
@ivanpauno 

>> name and namespace rules are applied in the order they appear, right?

> they order doesn't matter

the order of name and namespace rules doesn't matter.
I think users like to understand some rules by referring to an example description.
- A user gives the rules `talker:__ns:=/my_namespace` then `talker:__node:=foo`, or vice versa

@sloretz 
> They're still ordered within a rule, right?

I supposed you mean the order of multiple items about __node or __ns, the answer is yes.

> If I give talker:__node:=foo and talker:__node:=bar, then is the final name is foo?

Yes.
Actually, the example description already showed messages,
- talker's namespace is `/foo` because that rule was given first
though the above example uses `talker:__ns:=/foo` and `__ns:=/bar`

About this kind of information, I'll not update it in this PR.